docs/examples/State Estimation Example.ipynb

```diff
@@ -36,14 +36,18 @@
   },
   {
    "cell_type": "code",
-   "execution_count": 1,
+   "execution_count": 14,
```

Comment:
please restart kernel, clear output and rerun again to set the execution count back to `1`